### DISCUSSION TITLE
Allowing Scrolling Beyond Maximum or Minimum X Values

### DISCUSSION THREAD

**Discussion by czou40** [patrykandpatrick/vico#828] [Questions]
### Question

Hi, it seems that the charts can only scroll within the range between the minimum X value and the maximum X value of a dataset. Since my X values are dates, I want the chart to end at today's current date. Setting initialScroll to Scroll.Absolute.x(currentDate) doesn't seem to work, but setting initialScroll to a date between the earliest and latest dates in the dataset works fine.

Manual scrolling doesn't seem to work either. For example, July 27 is the latest date with data, and I can't scroll beyond July 27. Is there a way to disable this constraint to allow customized minX and maxX scrolling/zooming constraints or infinite scrolling/zooming?

![Screenshot_20240804-042618 1](https://github.com/user-attachments/assets/10bf9745-c96f-4e2a-afd3-d3fbaba8f82e)

### Vico version(s)

2.0.0-alpha.27

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hi @czou40
>Is there a way to disable this constraint to allow customized minX and maxX

Yes, there is. It's described in the [wiki](https://patrykandpatrick.com/vico/wiki/cartesian-charts/layers/#axisvalueoverrider).
